src/openrct2-ui/windows/Options.cpp

```diff
@@ -895,25 +909,11 @@ namespace OpenRCT2::Ui::Windows
             }
 
             // Disable Steam Overlay checkbox when using software or OpenGL rendering.
-            if (Config::Get().general.DrawingEngine == DrawingEngine::Software
-                || Config::Get().general.DrawingEngine == DrawingEngine::OpenGL)
-            {
-                disabled_widgets |= (1uLL << WIDX_STEAM_OVERLAY_PAUSE);
-            }
-            else
-            {
-                disabled_widgets &= ~(1uLL << WIDX_STEAM_OVERLAY_PAUSE);
-            }
+            // TODO: Check if this even works properly.
+            disabled_widgets |= (1uLL << WIDX_STEAM_OVERLAY_PAUSE);
```

Comment:
This now just blanket-disabled the checkbox, rather than enabling it for hardware display.